frontend/src/app/lightning/lightning-dashboard/lightning-dashboard.component.html

```diff
@@ -102,7 +102,6 @@ <h5 class="card-title d-inline" i18n="lightning.connectivity-ranking">Connectivi
     <a *ngIf="!officialMempoolSpace" [routerLink]="['/tx/push' | relativeUrl]" i18n="shared.broadcast-transaction|Broadcast Transaction">Broadcast Transaction</a>
   </div>
 
-  <br>
```

Comment:
Do we want to remove the last break row on the Lightning dashboard? The other dashboards have a break row in the bottom.

Reply:
Yes actually the lightning dashboard has two `<br>` at the bottom of the page. I removed the wrong one. Fixed.